src/main/java/bio/terra/service/dataset/DatasetService.java

```diff
@@ -188,9 +194,20 @@ public String ingestDataset(
             + ingestRequestModel.getTable()
             + " in dataset id "
             + id;
+    String computedLoadTag = loadService.computeLoadTag(ingestRequestModel.getLoadTag());
     return jobService
         .newJob(description, DatasetIngestFlight.class, ingestRequestModel, userReq)
         .addParameter(JobMapKeys.DATASET_ID.getKeyName(), id)
+        .addParameter(LoadMapKeys.LOAD_TAG, computedLoadTag)
+        .addParameter(
+            LoadMapKeys.DRIVER_WAIT_SECONDS,
+            configService.getParameterValue(ConfigEnum.LOAD_DRIVER_WAIT_SECONDS))
```

Comment:
The only reason its passed in is for inserting faults...

Reply:
I don't see any of these used for inserting faults. What I do see is them being extracted at the flight level and then converted to step parameters. But the code that does this also has access to `ConfigurationService`. What's more, the step that actually uses this data _also_ has access to `ConfigurationService` and could read it directly instead of using a flight map value.

Reply:
https://github.com/DataBiosphere/jade-data-repo/blob/develop/src/main/java/bio/terra/service/dataset/flight/ingest/IngestSetupStep.java